### DISCUSSION TITLE
Multiple Questions

### DISCUSSION THREAD

**Discussion by Newer1107** [prasanthrangan/hyprdots#60] [Q&A] [answered]
1) How do I set the power button of the pc to launch the dialogue where I can select what I want to do?
2) How do I adjust the display off timer?
3) During boot, sometimes anime girls shows up, how do I make sure that all the time anime girls show instead of the basic hyprland screen during boot?

**Comment by prasanthrangan** [comment, marked as answer]
Hi,

1. set `HandlePowerKey=ignore` in `/etc/systemd/logind.conf` and add this in `~/.config/hypr/keybindings.conf` -
```
bind = , XF86PowerOff, exec, wlogout -b 2 -c 0 -r 0 -L 930 -R 930 -T 300 -B 300 --protocol layer-shell # logout menu 
```

2. adjust the timeout value in `~/.config/hypr/hyprland.conf`, this line -
```
exec-once = swayidle -w timeout 900 'hyprctl dispatch dpms off' resume 'hyprctl dispatch dpms on' # turn off monitor after 15 mins
```

3. I'm not sure about this, the default grub theme is static.

**Comment by T3l3sc0p3** [comment]
@Newer1107 In the third question, look in the /usr/share/hyprland directory, you will see 2 anime girls that often appear during boot
One of them is Rimuru Tempest 😆
